go/pkg/constants/constants.go

```diff
@@ -29,24 +29,27 @@ var StablecoinDetails = map[int64]map[Stablecoin]StablecoinData{
 }
 
 type ConfigData struct {
-	VaultAddress  string
-	VaultDecimals int64
-	SDai          string
-	LiquidityPool string
+	VaultAddress   string
+	VaultDecimals  int64
+	AdapterAddress string
+	SDai           string
```

Comment:
to be renamed "Savings" in the whole codebase

Reply:
Even the functions `GetSDaiPrice` and `GetSDaiTotalValue` in `domains.go` ?
Even in `testCases` in `sdk_test.go` ?

Reply:
I didn't modifier those functions for now

Reply:
yep, better to rename everything. this sdk can be used both for sDAI and sUSDS, so we'd rather be token-agnositc everywhere